docs/operate/get-started/setup-micro.md

```diff
@@ -0,0 +1,66 @@
+---
+linkTitle: "Set up an ESP32"
+title: "Set up an ESP32"
+weight: 25
+layout: "docs"
+type: "docs"
+images: ["/installation/thumbnails/install.png"]
+imageAlt: "Install Viam"
+no_list: false
+description: "Install the lightweight version of the software that drives hardware and connects your device to the cloud."
+aliases:
+  - /installation/microcontrollers/
+  - /installation/prepare/microcontrollers/
+  - /installation/prepare/microcontrollers/
+  - /build/micro-rdk/
+  - /get-started/installation/microcontrollers/
+  - /installation/viam-micro-server-setup/
+---
+
+## Supported microcontrollers
+
+ESP32 microcontrollers must have at least 2 cores, 384kB SRAM, 2MB PSRAM and 8MB flash to work with Viam.
+The following microcontrollers have been tested with Viam:
+
+- [ESP32-WROVER Series](https://www.espressif.com/en/products/modules/esp32)
+
+## About ESP32 microcontroller setup
+
+Because microcontrollers do not run operating systems and are instead flashed with firmware, the setup process is different than for other devices running Viam.
```

Comment:
```suggestion
Because microcontrollers do not run operating systems and are instead flashed with firmware, the setup process is different than for regular computers and SBC's.
```

Reply:
```suggestion
Because microcontrollers do not run operating systems and are instead flashed with firmware, the setup process is different than for regular computers and SBCs.
```